dot/sync/syncer.go

```diff
@@ -270,7 +270,7 @@ func (s *Service) createBlockRequest(startInt int64) *network.BlockRequestMessag
 
 	blockRequest := &network.BlockRequestMessage{
 		ID:            randomID, // random
-		RequestedData: 3,        // block header + body
```

Comment:
could you declare `byte` consts for each of the possible data types and then `&` them together here?